### DISCUSSION TITLE
Hatch pattern "script" (from docs)

### DISCUSSION THREAD

**Discussion by mostl33** [mozman/ezdxf#1110] [General]
Hey!

Just a quick question: 

Is the following image from the documentation:

https://ezdxf.readthedocs.io/en/stable/_images/hatch-predefined-pattern.png

fron page:
https://ezdxf.readthedocs.io/en/stable/tutorials/hatch.html#load-hatch-patterns-from-file

created with via a "script"? Is the sourcecode available in the repository? 

Thanks in advance & best regards
ML

**Comment by mozman** [comment]
https://github.com/mozman/ezdxf/blob/master/examples_dxf/create_hatch_pattern_examples.py

This script creates a DXF with all pattern included in `ezdxf`. The rendering was done by a CAD application back then.
